# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1386003914 18000
#      Mon Dec 02 12:05:14 2013 -0500
# Branch stable
# Node ID 708173343c5059201b7ec7bdbd44aa5fee519f21
# Parent  c97ee13f1027bfd5edde3dfea269b4079d76a7ca
use "octave" instead of "\\s" in default prompt

* input.cc (set_default_prompts): Use "octave" instead of "\\s".
(FPS1): Fix doc string to match.

diff --git a/libinterp/corefcn/input.cc b/libinterp/corefcn/input.cc
--- a/libinterp/corefcn/input.cc
+++ b/libinterp/corefcn/input.cc
@@ -132,17 +132,20 @@ void
 remove_input_event_hook_functions (void)
 {
   input_event_hook_functions.clear ();
 }
 
 void
 set_default_prompts (void)
 {
-  VPS1 = "\\s:\\#> ";
+  // Use literal "octave" instead of "\\s" to avoid setting the prompt
+  // to "octave.exe" or "octave-gui", etc.
+
+  VPS1 = "octave:\\#> ";
   VPS2 = "> ";
   VPS4 = "+ ";
 
   octave_link::set_default_prompts (VPS1, VPS2, VPS4);
 }
 
 void
 octave_base_reader::do_input_echo (const std::string& input_string) const
@@ -1232,17 +1235,17 @@ for input.\n\
 DEFUN (PS1, args, nargout,
        "-*- texinfo -*-\n\
 @deftypefn  {Built-in Function} {@var{val} =} PS1 ()\n\
 @deftypefnx {Built-in Function} {@var{old_val} =} PS1 (@var{new_val})\n\
 @deftypefnx {Built-in Function} {} PS1 (@var{new_val}, \"local\")\n\
 Query or set the primary prompt string.  When executing interactively,\n\
 Octave displays the primary prompt when it is ready to read a command.\n\
 \n\
-The default value of the primary prompt string is @qcode{\"\\s:\\#> \"}.\n\
+The default value of the primary prompt string is @qcode{\"octave:\\#> \"}.\n\
 To change it, use a command like\n\
 \n\
 @example\n\
 PS1 (\"\\\\u@@\\\\H> \")\n\
 @end example\n\
 \n\
 @noindent\n\
 which will result in the prompt @samp{boris@@kremvax> } for the user\n\
